domains/authorization/Proxy/Program.cs

```diff
@@ -1,104 +1,118 @@
+using System.Reflection;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 using Asp.Versioning;
 using EnergyOrigin.Setup;
 using EnergyOrigin.TokenValidation.b2c;
 using EnergyOrigin.TokenValidation.Options;
 using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.OpenApi;
+using Microsoft.OpenApi.Extensions;
+using Proxy.Controllers;
 using Proxy.Options;
+using Swashbuckle.AspNetCore.Swagger;
 
-var builder = WebApplication.CreateBuilder(args);
-
-// Add services to the container.
-
-builder.Services.AddControllers();
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
-builder.Services.AddEndpointsApiExplorer();
+public class Program
+{
```

Comment:
minor comment - Not important. 
Do we need to wrap this in a public class Program???

Reply:
It is to get the args in the main function. Dno if there is another way?

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/main-command-line